### PR TITLE
Insta edits

### PR DESCRIPTION
**Pull request by lala-laura**
Instagram section pull request

### CODE REVIEW

**Comment by lala-laura** [comment]
Hold on, there's something I need to fix

File: assets/css/style.css [hunk 1]
```diff
@@ -1699,22 +1633,32 @@ grid-template-rows: repeat(2, 1fr);
...
+margin: -20px 0px 50px 0px;
+
+}
+.insta-name {
+  font-size: 55px;
+}
+
+.insta-hashtag{
+  margin: -40px 0px;
+  font-size: 45px;
```
Comment (loganmorganml1) [inline assets/css/style.css:1681]: I would keep this the same as the other h3 stylings, which would be 35px for mobile and tablet, and 55px for desktop. The desktop size looks fine but the size for other screen sizes need to go down (they are currently at 42px)

**Review by loganmorganml1 — CHANGES_REQUESTED** [review]
One edit, but looking good!

**Review by loganmorganml1 — APPROVED** [review]
Nothing to show, looks like it was merged cleanly!

**Pull request merged**